### DISCUSSION TITLE
Idea: Add scrubbing to frigate HA card

### DISCUSSION THREAD

**Discussion by SteveLobdell** [blakeblackshear/frigate#14236] [Ideas]
Trying to switch away from Google nest, the one thing that my wife likes is loading the app up, clicking a stream, and scrubbing is right there. Now I have to train her that to scrub she has to go to the frigate add on. That's the one thing I think the card is missing. First world problems, I know.

**Comment by hawkeye217** [comment]
This is the repo for Frigate itself, the card is maintained by another author and the repo is here: https://github.com/dermotduffy/frigate-hass-card
